File: Data/Monito.Database.Entities/Worker.cs
using System;
using System.Collections.Generic;
using Monito.Database.Entities.Interface;

namespace Monito.Database.Entities {
	public class Worker : IPrimaryKeyEntity, ITimestampTrackedEntity, IUUIDTrackedEntity {
		public int ID { get; set; }
		public string Hostname { get; set; }
		public Guid UUID { get; set; }
		public DateTime CreatedAt { get; set; }
		public DateTime UpdatedAt { get; set; }
		public virtual ICollection<Queue> Queues { get; set; }
	}
}

File: Presentation/Monito.Worker/models/worker.go
package models

import "time"

type Worker struct {
	ID        uint `gorm:"primary_key"`
	Hostname  string
	UUID      string `gorm:"type:varchar(100);unique_index"`
	Ping      time.Time
	CreatedAt time.Time
	UpdatedAt time.Time
	Queues    []Queue
}
